styleguide.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   },
   pagePerSection: true,
   require: [
+    '@babel/polyfill',
```

Comment:
Interesting, yeah, I think this makes sense .. we had this issue with the internal one for same reason until we did added a require in the styleguide config like:

`    './frontend/features/styleguide/polyfills.js',`

which imports the polyfills. Guess we can't reach that from here though, so this makes sense to me!